mycore-neo4j/src/main/resources/docker/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.5'
```

Comment:
These dockerfiles go into the mycore-neo4j.jar .
Is this intended? - Otherwise move to /docker  or src/docker or is there a default-location for docker files in Maven-projects?

Reply:
Talked to Sebastian, the files should NOT go into the JAR. 
There is no "default" location - please move to root folder or /docker.

Reply:
moved to mycoire-neo4j/docker